libs/viewer/src/SimpleViewer.cpp

```diff
@@ -418,34 +418,78 @@ void SimpleViewer::applyAnimation(double currentTime) {
     mAnimator->updateBoneMatrices();
 }
 
-void SimpleViewer::renderUserInterface(float timeStepInSeconds, View* guiView, float pixelRatio,
-        float mouseX, float mouseY, bool mouseButton, float mouseWheelY) {
+void SimpleViewer::renderUserInterface(float timeStepInSeconds, View* guiView, float pixelRatio) {
     if (mImGuiHelper == nullptr) {
         mImGuiHelper = new ImGuiHelper(mEngine, guiView, "");
 
+        auto& io = ImGui::GetIO();
+
+        // The following table assumes that keyCode arises from a web keydown event.
+        io.KeyMap[ImGuiKey_Tab] = 9;
```

Comment:
Could those keycodes be exposed as constants in the header file? What if we want to invoke the key handles from C++/Java? It seems the keycodes are just regular ASCII/ANSI codes so maybe document this in the key handler APIs.

Reply:
Yeah they're just ANSI codes. not really web-specific like the comment implies.  To address this I changed the wording and added a comment to the header file.